.github/workflows/python-tests.yaml

```
@@ -325,7 +325,7 @@ jobs:
       - name: Install packages
         run: |
           python -m pip install -U uv
-          uv pip install --upgrade --system -e .[dev] 'pydantic>=2.4,<3'
+          uv pip install --upgrade --system -e .[dev] 'pydantic>=2.4,<3' 'pydantic_settings>=2.2,<3'
```

https://github.com/pydantic/pydantic-settings/releases/tag/v2.2.1
```suggestion
          uv pip install --upgrade --system -e .[dev] 'pydantic>=2.4,<3' 'pydantic_settings>=2.2.1,<3'
```